UI/window-basic-main-outputs.cpp

```diff
@@ -1984,8 +1984,25 @@ inline void AdvancedOutput::SetupFFmpeg()
 		config_get_int(main->Config(), "AdvOut", "FFAEncoderId");
 	const char *aEncCustom =
 		config_get_string(main->Config(), "AdvOut", "FFACustom");
+
+	OBSDataArrayAutoRelease audio_names = obs_data_array_create();
+
+	for (size_t i = 0; i < MAX_AUDIO_MIXES; i++) {
+		string cfg_name = "Track";
```

Comment:
As the identifier is used for the configuration file only, there's no need to make this human readable.

`audioTrackName<NUMBER>` would be easier to compose and can start at `0` (though admittedly my alternative is _also_ more human readable).

Reply:
Track<N>Name is how the values are stored in the profile config:
```
[AdvOut]
Track1Name=Main
Track2Name=Mic
Track3Name=Desktop
Track4Name=SomeName4
Track5Name=SomeName5
Track6Name=SomeName6
```

The string built here is used just below to fetch the value from the profile config file.

Reply:
So that's on us then, too bad. 🙈

Reply:
Indeed. We may wish to change the name at some point in the future, but that's out of scope for this PR.